src/ResourceManager/Compute/Commands.Compute/help/Start-AzVM.md

```diff
@@ -21,7 +21,7 @@ Start-AzVM [-Name] <String> [-ResourceGroupName] <String> [-AsJob] [-DefaultProf
 
 ### IdParameterSetName
 ```
-Start-AzVM [-Name] <String> [-Id] <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+Start-AzVM [[-Name] <String>] [-Id] <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
```

Comment:
If you specify both name and Id, the name specified will be used rather than the one parsed from the Id (to prevent breaking changes for customers). I’ll change one of the tests above to show that scenario.